src/lib/redis-cache/RedisCacheClient.ts

```diff
@@ -0,0 +1,44 @@
+import { Result } from '@sapphire/result';
+import { isNullish } from '@sapphire/utilities';
+import { envParseInteger, envParseString } from '@skyra/env-utilities';
+import { container } from '@skyra/http-framework';
+import Redis from 'ioredis';
```

Comment:
Wait, it's imported from default and not from `{ Redis }`? Or is the latter type-only but incorrectly exported as a class instead of a type?